nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol

```diff
@@ -244,36 +251,34 @@ contract PolicyManager is Upgradeable {
             } else if (endPeriod < minPeriod) {
                 continue;
             }
-            downtimePeriods = downtimePeriods.add16(
-                AdditionalMath.min16(maxPeriod, endPeriod)
-                .sub16(AdditionalMath.max16(minPeriod, startPeriod))
-                .add16(1));
+            downtimePeriods += AdditionalMath.min16(maxPeriod, endPeriod)
+                .sub16(AdditionalMath.max16(minPeriod, startPeriod)) + 1;
```

Comment:
There's still a `sub16()` in here. Is that intentional?

Reply:
It's intentional, mostly for unreal situation when `getPastDowntime` will return something unpredictable